File: java/adhan/src/main/java/com/batoulapps/adhan/Coordinates.java
package com.batoulapps.adhan;

/**
 * Coordinates representing a particular place
 */
public class Coordinates {

  /**
   * Latitude
   */
  public final double latitude;

  /**
   * Longitude
   */
  public final double longitude;

  /**
   * Generate a new Coordinates object
   * @param latitude the latitude of the place
   * @param longitude the longitude of the place
   */
  public Coordinates(double latitude, double longitude) {
    this.latitude = latitude;
    this.longitude = longitude;
  }
}

File: Swift/Sources/Enums/Coordinates.swift
//
//  Coordinates.swift
//  Adhan
//
//  Copyright © 2018 Batoul Apps. All rights reserved.
//

import Foundation

/* Latitude and longitude */
public struct Coordinates {
    let latitude: Double
    let longitude: Double

    public init(latitude: Double, longitude: Double) {
        self.latitude = latitude
        self.longitude = longitude
    }
}
